workspaces/redhat-argocd/.changeset/nervous-tigers-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-redhat-argocd': major
```

Comment:
I am not sure if this has to be a major release.  can we make it minor release ?

Reply:
Changed to minor release